tools/README.md

```diff
@@ -1,30 +1,31 @@
 # Boutiques Tools
 Tools to create and use boutiques descriptors.
 
-Note: there is a Docker image ([boutiques/boutiques](https://hub.docker.com/r/boutiques/boutiques)) with the tools installed.
+## Installation
 
-1. `validator.rb`: a Ruby script to validate Boutiques descriptors. Requires Ruby >= 2.0.
-  * Installation:
-    * `gem install bundler`
-    * `bundle install`
-  * Usage:
-    * `validator.rb ../schema/descriptor.schema.json ./tool.json`
-  * Usage example with Docker image:
-    * `docker run --rm -v $PWD:/work -w /work boutiques/boutiques validator.rb /usr/local/boutiques/schema/descriptor.schema.json ./tool.json`
+* pip install boutiques
 
+## Usage
+
+1. `validator.py`: a validator of Boutiques descriptors.
```

Comment:
After pip installing, `validator.py` is accessible everywhere, not just within this directory, as `bosh-validate`

So, change `validator.py` -> `bosh-validate`?